taskcluster/scripts/toolchain/bundle_qt_ios.sh

```diff
@@ -5,8 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 set -x
 
-
```

Comment:
this is a random change :)

Reply:
its intentional :) - 

changeing the `env:` vom 2.3 to 2.4 will not invalidate the old toolchain task. It's only updated if anything it depends on (taskcluster-version, any task it depends on, or changes in bundle_qt_ios.sh ) changes. Therefore as we landed the patch initially, we did not get a new task that generates that. 

I'm currently working on expressing those types of things - see #3999